Realm/RLMMongoCollection.h

```diff
@@ -0,0 +1,264 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import <Foundation/Foundation.h>
+#import "RLMObjectId.h"
+
+NS_ASSUME_NONNULL_BEGIN
+@protocol RLMBSON;
+
+@class RLMApp, RLMFindOptions, RLMFindOneAndModifyOptions, RLMUpdateResult;
+
+/**
+* The `RLMMongoCollection` represents a MongoDB collection.
+*
+* You can get an instance from a `RLMMongoDatabase`.
+*
+* Create, read, update, and delete methods are available.
+*
+* Operations against the Realm Cloud server are performed asynchronously.
+*
+* - Note:
```

Comment:
This isn't the correct syntax for this in obj-c  and it'll end up an unreadable mess in both the alt-click window in Xcode and in the API docs.